ruby_on_rails/apis/apis_and_building_your_own.md

```diff
@@ -93,13 +93,14 @@ In our case, we'll do this by modifying `#as_json` in our model to return only t
   class User < ActiveRecord::Base
 
     # Option 1: Purely overriding the #as_json method
-    def as_json(options={})
+    def as_json(_options={})
       { :name => self.name }  # NOT including the email field
     end
 
     # Option 2: Working with the default #as_json method
     def as_json(options={})
-      super(:only => [:name])
+      options = { only: [:name] }.merge(options)
```

Comment:
Using options as the name of the argument and the temp variable is confusing. Is there a better name we can use for the temp variable?

Or maybe we can inline it?
```suggestion
      super({ only: [:name] }.merge(options))
```

Reply:
Fair point! I am happy with inlining, could not think of a better name that was less confusing.